src/lib/utils.ts

```diff
@@ -584,6 +584,31 @@ export function getFromLookup<V>(value: unknown, lookup: {[s: number | string]:
     return result ?? defaultValue;
 }
 
+// Silly hack, but boolean is not supported as index
+export function getFromLookupBool(value: unknown, lookup: {[s: string | number]: string | number}): boolean | number {
```

Comment:
Instead of adding a new function, could we maybe extend `getFromLookup` like this: 

```
export function getFromLookup<V>(value: unknown, lookup: {[s: number | string]: V}, defaultValue: V=undefined, keyIsBool: boolean=false): V {
```

Reply:
Merged them together